### PR TITLE
[Docs] Glossary - objects & controls listed

### PR DESCRIPTION
**Pull request by seaniepie**
- Add a list of objects and controls to glossary/object.lcdoc
- Controls laid out in bullets and 'widget' added to list in glossary/control.lcdoc

### CODE REVIEW

File: docs/glossary/o/object.lcdoc [hunk 1]
```diff
@@ -6,9 +6,15 @@ Type: glossary
 
 Description:
 A LiveCode module capable of receiving <message|messages>. One of the
-building blocks of a LiveCode application.
+building blocks of a LiveCode application. Objects include <stack|stacks>, 
+<card|cards> and <control|controls>. <control|Controls> include 
+<button|buttons>, <field|fields>, <image|images>, <graphic|graphics>, 
+<player|players>, <EPS|EPS objects>, <scrollbar|scrollbars>, and <group|groups>.
```
Comment (livecodeali) [inline docs/glossary/o/object.lcdoc]: @seaniepie You've missed widget from this list

**Comment by seaniepie** [comment]
In the last post you said _A widget is a control, so it should probably be added to that list and not as an explicit example of an object._ so I took that to mean to not include it here. I guess the key word not emphasised enough was **explicit**.

**Comment by livecodeali** [comment]
@seaniepie Ah, sorry for the confusion - in fact I was suggesting that the controls list not be included in the object glossary at all, and just be in the control glossary 😄  I don't see any particular harm in having it in both places. But maybe it should be a list in both... what do you think?

**Comment by seaniepie** [comment]
I personally would like it there as I can never seem to remember the word 'control' when thinking of objects. Especially in the context of trying to search through objects on a card that have a particular setting. I know it begins with a c but beyond that I have an odd mental block :/ so it would be nice to just look up 'object' and it has reference to controls as a little reminder and nudge. We used to have in the old dictionary a 'similar' section. I should think that references serve a _similar_ function. :)

**Comment by livecodeali** [comment]
@seaniepie I think it'd be worth formatting it as a list in the object glossary too then

**Comment by seaniepie** [comment]
Having it as lists in both is an excellent idea. Thanks Ali.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 49d1bc7
@seaniepie Thank you!

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 49d1bc798d212f83c42deb497ff9478b53b0c241

**Comment by livecode-vulcan** [comment]
😎 test success 49d1bc798d212f83c42deb497ff9478b53b0c241
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/431)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/436)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/431)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/431)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/437)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/431)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/431)

**Pull request merged**